Homework_3.py

```diff
@@ -0,0 +1,46 @@
+# import os
+# import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+# Making spark object
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+# creating connection to Mariadb
+database_name = "baseball"
+port = "3306"
+user = "root"
```

Comment:
Comment to be discarded: 

The problem with specifying hardcoded user is that mysql stops working from command line with any other user. I had to explicitly add the privileges in mysql for baseball database.